ansible_collections/arista/cvp/docs/how-to/v3/cv_change_control_v3.md

```diff
@@ -18,9 +18,9 @@
   - `state: approve`: Approve Change control
   - `state: unapprove`: Unpprove Change control
   - `state: execute`: Execute Change control
-  - `state: schedule`: Schedule Change control
+  - `state: schedule`: Schedule Change control (available only on cvp2022.1.0 and newer)
   - `state: approve_and_execute`: Approve and Execute Change control
-  - `state: schedule_and_approve`: Schedule and Approve Change control
+  - `state: schedule_and_approve`: Schedule and Approve Change control (available only on cvp2022.1.0 and newer)
```

Comment:
can we change both of the above to `(available only on CVP 2022.1.0 and newer or CVaaS)`